beacon-chain/sync/subscriber_committee_index_beacon_attestation.go

```diff
@@ -0,0 +1,28 @@
+package sync
+
+import (
+	"context"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+)
+
+func (r *RegularSync) committeeIndexBeaconAttestationSubscriber(ctx context.Context, msg proto.Message) error {
+
+	return nil
+}
+
+func (r *RegularSync) currentCommitteeIndex() int {
+	state, err := r.chain.HeadState(context.Background())
+	if err != nil {
+		panic(err)
+	}
+	if state == nil {
+		return 0
+	}
+	count, err := helpers.CommitteeCountAtSlot(state, helpers.StartSlot(helpers.CurrentEpoch(state)))
```

Comment:
Need advance state to current slot in case it falls behind

Reply:
If there are skip slots, there would be no committee changes? What is the risk of not advancing this?

Reply:
Discussed offline, this method only runs on a state feed update anyway so there is no risk of skip slots during this method call.

Reply:
Skip slot will cause committee count to decrease but wont increase unless there's finality. Given this is based on state feed (i.e called during state changes) It's fine to use head state here